### PR TITLE
Update office-365-service-encryption.md

### PR DESCRIPTION
**Pull request by msbemba**
Corrected SFB to Teams on this document.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3869

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/office-365-service-encryption.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ description: "Summary: Understand data resiliency in Microsoft Office 365."
 
 # Service Encryption
 
-In addition to using volume-level encryption, Exchange Online, Skype for Business, SharePoint Online, and OneDrive for Business also use Service Encryption to encrypt customer data. Service Encryption allows for two key management options:
+In addition to using volume-level encryption, Exchange Online, Teams, SharePoint Online, and OneDrive for Business also use Service Encryption to encrypt customer data. Service Encryption allows for two key management options:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/office-365-service-encryption.md]:
```suggestion
In addition to using volume-level encryption, Exchange Online, Microsoft Teams, SharePoint Online, and OneDrive for Business also use Service Encryption to encrypt customer data. Service Encryption allows for two key management options:
```

File: microsoft-365/compliance/office-365-service-encryption.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ Using Customer Key, you can generate your own cryptographic keys using either an
 
 Another benefit of Customer Key is the control you have over the ability of Microsoft to process your data. If you want to remove data from Office 365, such as if you want to terminate service with Microsoft or remove a portion of your data stored in the cloud, you can do so and use Customer Key as a technical control. Removing data ensures that no one, including Microsoft, can access or process the data. Customer Key is in addition and complementary to Customer Lockbox that you use to control access to your data by Microsoft personnel.
 
-To learn how to set up Customer Key for Microsoft 365 for Exchange Online, Skype for Business, SharePoint Online, including Team Sites, and OneDrive for Business, see these articles:
+To learn how to set up Customer Key for Microsoft 365 for Exchange Online, Teams, SharePoint Online, including Team Sites, and OneDrive for Business, see these articles:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/office-365-service-encryption.md]:
```suggestion
To learn how to set up Customer Key for Microsoft 365 for Exchange Online, Microsoft Teams, SharePoint Online, including Team Sites, and OneDrive for Business, see these articles:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**